### PR TITLE
Adds initial support for Butterknife

### PR DESCRIPTION
**Pull request by goober**

### CODE REVIEW

File: app/src/main/java/com/liato/bankdroid/MainActivity.java [hunk 1]
```diff
@@ -145,15 +153,7 @@ public void onResume() {
...
-        if (banks.size() > 0) {
-            findViewById(R.id.txtAccountsDesc).setVisibility(View.GONE);
-//            showTitleButton("refresh");
-            //findViewById(R.id.btnAccountsRefresh).setClickable(true);
-        } else {
-            findViewById(R.id.txtAccountsDesc).setVisibility(View.VISIBLE);
-//            hideTitleButton("refresh");
-            //findViewById(R.id.btnAccountsRefresh).setClickable(false);
-        }
+        mAccountsDescription.setVisibility(banks.isEmpty() ? View.VISIBLE : View.GONE);
```
Comment (fredrike) [inline app/src/main/java/com/liato/bankdroid/MainActivity.java]: WOW!  :+1:

File: app/src/main/java/com/liato/bankdroid/BankEditActivity.java [hunk 2]
```diff
@@ -116,21 +140,22 @@ public void onCreate(Bundle savedInstanceState) {
...
 
     @Override
+    @OnClick({R.id.btnSettingsCancel, R.id.btnSettingsOk})
     public void onClick(View v) {
         if (v.getId() == R.id.btnSettingsCancel) {
             this.finish();
         } else if (v.getId() == R.id.btnSettingsOk) {
             SELECTED_BANK.setUsername(
-                    ((EditText) findViewById(R.id.edtBankeditUsername)).getText().toString()
+                    mUsernameField.getText().toString()
```
Comment (fredrike) [inline app/src/main/java/com/liato/bankdroid/BankEditActivity.java]: Doesn't getText return a string?

Comment (goober) [inline app/src/main/java/com/liato/bankdroid/BankEditActivity.java]: Unfortunately `getText()` returns an `Editable` and not a `String`.

**Comment by fredrike** [comment]
Thank you!

File: app/src/main/java/com/liato/bankdroid/BankEditActivity.java (hunk 2 above)
Comment (fredrike) [inline app/src/main/java/com/liato/bankdroid/BankEditActivity.java]: Ok..

**Comment by goober** [comment]
Closes #330

**Pull request merged**